package.json

```
@@ -70,18 +70,20 @@
     "build:win": "grunt win-prod",
     "bundle:dev": "webpack",
     "bundle": "webpack --env.production",
-    "edge": "npm run prestart && electron electron --inspect --devtools --enable-logging --env=https://wire-webapp-edge.zinfra.io",
     "fix:js": "npm run test:js -- --fix",
     "fix:other": "npm run prettier -- --write",
     "fix": "npm run fix:js && npm run fix:other",
     "jest": "jest",
-    "localhost": "npm run prestart && electron electron --inspect --devtools --enable-logging --env=http://localhost:8080/app",
     "postinstall": "cd electron && electron-builder install-app-deps",
     "prestart": "npm run bundle:dev",
     "prettier": "prettier \"**/*.{json,md,css}\"",
-    "prod": "npm run prestart && electron electron --inspect --devtools --enable-logging --env=https://app.wire.com",
-    "staging": "npm run prestart && electron electron --inspect --devtools --enable-logging --env=https://wire-webapp-staging.zinfra.io",
-    "start": "electron electron --inspect --devtools --enable-logging",
+    "start:dev": "npm run prestart && electron electron --inspect --devtools --enable-logging --env=https://wire-webapp-dev.zinfra.io",
+    "start:edge": "npm run prestart && electron electron --inspect --devtools --enable-logging --env=https://wire-webapp-edge.zinfra.io",
+    "start:internal": "npm run prestart && electron electron --inspect --devtools --enable-logging --env=https://wire-webapp-staging.wire.com/?env=prod",
+    "start:localhost": "npm run prestart && electron electron --inspect --devtools --enable-logging --env=http://localhost:8080/app",
+    "start:prod": "npm run prestart && electron electron --inspect --devtools --enable-logging --env=https://app.wire.com",
+    "start:staging": "npm run prestart && electron electron --inspect --devtools --enable-logging --env=https://wire-webapp-staging.zinfra.io",
```

This should work as well..

```suggestion
    "start:staging": "npm run start -- --env=https://wire-webapp-staging.zinfra.io",
```

and the same for the rest..